docs/how-tos/create-custom-training-scripts.md

```diff
@@ -430,6 +431,33 @@ async def connect() -> ViamClient:
 {{% /tablestep %}}
 {{< /table >}}
 
+## Test your training script locally
+
+You can export one of your Viam datasets to test your training script locally.
+
+{{< table >}}
+{{% tablestep %}}
+**1. Export your dataset**
+
+You can get the dataset id from the dataset page or using the [`viam dataset list`](/cli/#dataset) command:
+
+```sh {class="command-line" data-prompt="$" data-output="1-10"}
+viam dataset export --destination=<destination> --dataset-id=<dataset-id> --include-jsonl=true
+```
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Run your training script locally**
+
+Install any required dependencies and run your training script specifying the path to the <dataset.jsonl> file from your exported dataset:
+
+```sh {class="command-line" data-prompt="$" data-output="1-10"}
+python3 -m model.training --dataset_file=/path/to/dataset.jsonl --model_output_directory=.
```

Comment:
Is it worth mentioning that the dataset file will be formatted like the one Viam produces for training and reference that the parsing functions (`parse_filenames_and_labels_from_json` and `parse_filenames_and_bboxes_from_json`) should be used to get the images and annotations from the dataset file?